### DISCUSSION TITLE
Best.pt and last.pt

### DISCUSSION THREAD

**Discussion by Zzwww09** [ultralytics/yolov5#5393] [Q&A]
Hello, what is the standard of best.pt ? or how to determine whether it is best.pt and which indicators are passed? In addition, when I predict, sometimes last.pt is better than best.pt 。Looking forward to your answer.

**Comment by glenn-jocher** [comment]
@Zzwww09 best.pt is saved on every maximum fitness epoch. Fitness is computed by a weighted combination of metrics:
https://github.com/ultralytics/yolov5/blob/ed887b5976d94dc61fa3f7e8e07170623dc7d6ee/utils/metrics.py#L15-L19

**Reply by Zzwww09** [reply]
Thank you for your reply.

**Comment by Audrey528** [comment]
@glenn-jocher Why did you choose fitness as indicator? I am confused, looking forward to your reply!

**Reply by glenn-jocher** [reply]
Fitness is computed by a weighted combination of metrics:
https://github.com/ultralytics/yolov5/blob/ed887b5976d94dc61fa3f7e8e07170623dc7d6ee/utils/metrics.py#L15-L19
